README.md

```diff
@@ -4,90 +4,90 @@ minideb
+  * The image is based on glibc for wide compatibility, and has apt for access to a large number of packages. In order to reduce size of the image, some packages that aren't required in containers are removed:
+    * Packages that aren't often used in containers (hardware related, init systems etc.)
+    * Some files that aren't usually required (docs, man pages, locales, caches)
+  * These images also include an `install_packages` command that you can use instead of apt. This takes care of some things for you:
+    * Install the named packages, skipping prompts etc.
+    * Clean up the apt metadata afterwards to keep the image small.
+    * Retrying if apt fails. Sometimes a package will fail to download due to a network issue, and this may fix that, which is particularly useful in an automated build pipeline.
+
+    For example:
+    ```
     $ install_packages apache2 memcached
+    ```
 
-The minideb container image is the base image for many Bitnami-maintained
-language runtimes including 
-[php](https://github.com/bitnami/bitnami-docker-php-fpm), 
-[nodejs](https://github.com/bitnami/bitnami-docker-node), 
-[ruby](https://github.com/bitnami/bitnami-docker-ruby) 
-and infrastructure components
-including [mariadb](https://github.com/bitnami/bitnami-docker-mariadb), 
-[redis](https://github.com/bitnami/bitnami-docker-redis), 
-[nginx](https://github.com/bitnami/bitnami-docker-nginx) and 
-[mongodb](https://github.com/bitnami/bitnami-docker-mongodb). 
+# Adoption of Minideb
+The minideb container image is the base image for many Bitnami-maintained language runtimes including
+  [php](https://github.com/bitnami/bitnami-docker-php-fpm),
+  [nodejs](https://github.com/bitnami/bitnami-docker-node),
+  [ruby](https://github.com/bitnami/bitnami-docker-ruby)
+  and infrastructure components
+  including [mariadb](https://github.com/bitnami/bitnami-docker-mariadb),
+  [redis](https://github.com/bitnami/bitnami-docker-redis),
+  [nginx](https://github.com/bitnami/bitnami-docker-nginx) and
+  [mongodb](https://github.com/bitnami/bitnami-docker-mongodb).
 
-Compatibility
--------------
+# Compatibility
+The image points to the Debian archive, so you are free to install packages from there that you need. However because some `Essential` packages have been removed they may not always install or work correctly.
 
-The image points to the Debian archive, so you are free to
-install packages from there that you need. However because
-some `Essential` packages have been removed they may not
-always install or work correctly.
-
-In those cases you can figure out which package is needed
-and manually specify to install it along with your desired
-packages.
-
-Docker
-------
-
-You can use the image directly, e.g.
+In those cases you can figure out which package is needed and manually specify to install it along with your desired packages.
```

Comment:
We ask users to create an issue request so that we can help them with such issues.